candi/bashible/common-steps/all/003_add_deckhouse_path.sh.tpl

```diff
@@ -15,3 +15,18 @@
 bb-sync-file /etc/profile.d/02-deckhouse-path.sh - << "EOF"
 export PATH="/opt/deckhouse/bin:$PATH"
 EOF
+
+# On Alt Linux there is condition
+# for f in /etc/profile.d/*.sh; do
+#        if [ -f "$f" -a -r "$f" -a -x "$f" -a -s "$f" -a ! -L "$f" ]; then
+#                . "$f"
+#        fi
+# done
+# so /etc/profile.d/02-deckhouse-path.sh should be executable
+chmod a+x /etc/profile.d/02-deckhouse-path.sh
+
+# also need to sure that path to deckhouse bin in root bashrc
+if grep -q "^PATH=" /root/.bashrc 2>/dev/null; then
+  sed 's/^PATH=.*//' -i /root/.bashrc
+fi
+echo "export PATH=/opt/deckhouse/bin:$PATH" >> /root/.bashrc
```

Comment:
This command will run every 4 hours.

Reply:
And file /root/.bashrc will increase size every 4 hours